windows/cross-compile/build.sh

```diff
@@ -106,11 +106,12 @@ cp -r /export/* "$QTOX_PREFIX_DIR"
 # Run tests
 set +u
 if [[ $RUN_TESTS -ne 0 ]]; then
-  export WINEPATH='/export;/windows/bin'
+  export WINEQT_QPA_PLATFORM='offscreen'
+  export WINEQT_PLUGIN_PATH='z:\export'
+  export WINEPATH='z:\export;z:\windows\bin'
   export CTEST_OUTPUT_ON_FAILURE=1
   export PATH="$PATH:/opt/wine-stable/bin"
-  # TODO(iphydf): Fix tests on windows.
-  # ctest -j$(nproc)
+  ctest -j"$(nproc)"
```

Comment:
--output-on-failure

Reply:
Shouldn't make a difference vs. the envvar but sure. I agree it's nicer this way.